### DISCUSSION TITLE
nexrender-action-hiec-convert

### DISCUSSION THREAD

**Discussion by JamesBotterill** [inlife/nexrender#842] [Show and tell]
I'm working on a small module that will check the extension of an image and convert it to png if it's an hiec image. 

It's still a work in progress but thought I'd share

https://github.com/JamesBotterill/nexrender-action-hiec-convert

**Comment by inlife** [comment]
Awesome. Feel free to add it to the readme section with external packages when you feel its finished! :tada:
